packages/appstore/src/components/account-manager/account-manager.tsx

```diff
@@ -92,16 +92,9 @@ const AccountManager = ({
                         </Button>
 
                         {platform === CFD_PLATFORMS.DXTRADE && (
```

Comment:
Why do you keep this condition if both platforms are opening the same modal. Looks like a duplicated code to me.